### ISSUE TITLE
Add incrementing/decrementing for break/session lengths

### ISSUE THREAD

**Issue by camchardukian** [camchardukian/fcc-react-typescript-pomodoro-clock#5]
Implement incrementing/decrementing such that the following units tests from FCC pass:
```
User Story #12: When I click the element with the id of break-decrement, the value within id="break-length" decrements by a value of 1, and I can see the updated value.

User Story #13: When I click the element with the id of break-increment, the value within id="break-length" increments by a value of 1, and I can see the updated value.

User Story #14: When I click the element with the id of session-decrement, the value within id="session-length" decrements by a value of 1, and I can see the updated value.

User Story #15: When I click the element with the id of session-increment, the value within id="session-length" increments by a value of 1, and I can see the updated value.
```

**Comment by camchardukian** [comment]
Incrementing and decrementing was implemented in PR #6. For some reason, however, most of the unit tests do not pass despite manual testing clearly showing the buttons are working as intended.

For that reason, we will focus on other issues for now as it seems our unit tests inability to pass may be related to our app not behaving as expected in other regards due to such a limited amount of the app having thus far been implemented.

We can revisit this issue in the next week or two if this issue persists after the other features have also been implemented.

**Comment by camchardukian** [comment]
As of July 2nd, 2020, these unit tests all pass in the latest version of our app.
